test/e2e_node/oomkiller_linux_test.go

```diff
@@ -265,3 +268,44 @@ func getOOMTargetContainerWithoutLimit(name string) v1.Container {
 		},
 	}
 }
+
+var _ = SIGDescribe("OOMKiller with LimitedSwap", framework.WithSerial(), framework.WithFeature("LimitedSwap"), func() {
```

Comment:
```suggestion
var _ = SIGDescribe("OOMKiller with LimitedSwap", framework.WithSerial(), framework.WithFeature("Swap"), func() {
```

Reply:
Or something similar to this. You can only reference existing features with this.